ibis/pandas/core.py

```diff
@@ -384,3 +466,47 @@ def execute_and_reset(
     elif isinstance(result, pd.Series):
         return result.reset_index(drop=True)
     return result
+
+
+compute_time_context = Dispatcher(
+    'compute_time_context',
+    doc="""\
+
+Compute time context for a node in execution
+
+Notes
+-----
+For a given node, return with a list of timecontext that are going to be
+passed to its children nodes.
+time context is useful when data is not uniquely defined by op tree. e.g.
+a TableExpr can represent the query select count(a) from table, but the
+result of that is different with time context (20190101, 20200101) vs
```

Comment:
I would show these as strings e.g. "('20190101', '20200101')" as ints work but in reality are prohibited as they do funny things
```
In [220]: pd.to_datetime(20200207)                                                                                                                             
Out[220]: Timestamp('1970-01-01 00:00:00.020200207')
```

Reply:
I agree, prohibited it in supported timecontext types.